BEIMA.Client/src/services/AddBuilding.js

```diff
@@ -7,9 +7,9 @@ const API_URL = process.env.REACT_APP_API_URL;
  * @param The user details to add to the DB
  * @return Error message or the inserted building ID
  */
-export default async function addBuilding(buildingDetails) {
+export default async function addBuilding(buildingDetails, token) {
```

Comment:
This kinda feels bad having to pass the token in every time, plus I think this breaks all of the logic that calls these functions currently. Is there possibly a way for us to store this token in some sort of global variable that we set on login?

Reply:
Actually, it doesn't break it. I thought it would too, but it gets ignored by the backend since the endpoints don't check for it yet. 

In my other PR, I created a useState Constant in App.js to store the currentUser, which will hold the token. My idea was when we make these service calls, then we retrieve the token from the currentUser variable and pass it to the function. I could change it to use the useOutlet and grab the token from the currentUser variable

Reply:
I like that approach better, it removes the redundancy for us to have to remember to pass the token in every time we want to hit an endpoint.

Reply:
We may have to pass the token as a parameter. I updated them to use the useOutletContext, but since these functions aren't components, it doesn't like it. I will have to update my other auth service function as well 
![image](https://user-images.githubusercontent.com/58532471/162637830-fcb5fd5f-5662-408b-b8f6-bd8504797287.png)

Reply:
maybe we can define a function that returns the token that we can call?

Reply:
I think I can do that in App.js and put it in the AppLayout component. I will play around with it some more

Reply:
Okay, I have changes up that use a function call to the auth service to get the token rather than passing it in as a parameter